expr_test.go

```diff
@@ -218,3 +264,149 @@ func ExampleEval_marshal() {
 
 	// Output: 3
 }
+
+func TestExpr(t *testing.T) {
+	tests := []struct {
+		name    string
+		code    string
+		env     interface{}
+		request interface{}
+		want    interface{}
+		wantErr bool
+	}{
+		{
+			name: "+ operator",
+			code: "1 + 1",
+			want: 2,
+		},
+		{
+			name:    "associativity",
+			code:    "(A * B) * C == A * (B * C)",
+			env:     struct{ A, B, C int }{},
+			request: struct{ A, B, C int }{A: 1, B: 2, C: 3},
```

Comment:
Here is to much boilerplates to write. What about defining it only once and just checking expressions? Take a llot at `vm` package tests.